drivers/gles3/shaders/scene.glsl

```diff
@@ -1070,15 +1066,11 @@ void main() {
 		fog = fog_process(vertex);
 	}
 #endif // !DISABLE_FOG
-#endif //!CUSTOM_FOG_USED
+#endif // !CUSTOM_FOG_USED
 
 	uint fog_rg = packHalf2x16(fog.rg);
 	uint fog_ba = packHalf2x16(fog.ba);
 
-#endif //!MODE_RENDER_DEPTH
-
-#ifndef MODE_RENDER_DEPTH
```

Comment:
There might have been code between those two at some point, but right now it's just closing to reopen the same scope.